src/constants.py

```diff
@@ -20,7 +20,7 @@
 
 JDBC_PORT = 10009
 
-KYUUBI_OCI_IMAGE = "ghcr.io/canonical/charmed-spark:3.4-22.04_edge"
+KYUUBI_OCI_IMAGE = "ghcr.io/canonical/charmed-spark-kyuubi:3.4-22.04_edge"
```

Comment:
suggestion: can we get this value from `metadata.yaml` somehow so that we have only one place where we specify the image (i.e. metadata.yaml) and everything else works out of the box?